frontend/packages/kubevirt-plugin/integration-tests/tests/models/importWizard.ts

```diff
@@ -43,57 +43,111 @@ export class ImportWizard extends Wizard {
     await click(view.importButon);
   }
 
+  async importNetworkStep(networkResources: Network[]) {
+    // Binding networks
+    // First update imported network interfaces to comply with k8s
+    await this.updateImportedNICs();
+    // Adding networks if any
+    if (networkResources) {
+      await this.addVmNetworks(networkResources);
+    }
+    await this.next();
+  }
+
+  async importDiskStep(storageResources: Disk[]) {
+    // Binding storage disks
+    // First update disks that come from the source VM
+    await this.updateImportedDisks();
+    // Adding disks if any
+    if (storageResources) {
+      await this.addVmStorage(storageResources);
+    }
+    await this.next();
+    // CloudInit page is in read-only mode
+    await this.next();
```

Comment:
disk/storage step should not include advanced step handling

Reply:
fixed, did that by mistake